econsa/correlation.py

```diff
@@ -194,16 +187,39 @@ def _special_dist(distributions):
         return False
 
 
-def _criterion(rho_c, arg, distributions, num_points, rule):
+def _criterion_gh(rho_c, arg, distributions, order=15, rule="gaussian"):
```

Comment:
Why not just remove `rule` altogether ... including docstring.

Reply:
.. please rewrite whole docstring, We are not interested in the changes now but what the function actually takes as inputs.

Reply:
... please also add a the folling as a proper reference. https://www.elsevier.com/books/methods-of-numerical-integration/davis/978-0-12-206360-2 ... make sure it also shows up properly in our docs.

Reply:
Is the book the same one as this? That was the original one you sent me. https://books.google.de/books?id=9Wxk_z9HskAC